packages/documentation-framework/templates/mdx.js

```diff
@@ -60,9 +65,19 @@ const MDXChildTemplate = ({
           To learn more about the process, visit our <Link to="/get-started/about#beta-components">about page</Link> or our <a href="https://github.com/patternfly/patternfly-org/tree/main/beta-component-promotion">Beta components</a> page on GitHub.
         </InlineAlert>
       )}
+      {(deprecated || source === 'react-deprecated') && (
+        <InlineAlert title="Deprecated feature" variant="warning">
+          This implementation has been deprecated in favor of a newer implementation, and is no longer getting maintained or enhanced.
+          {newImplementationLink && (
```

Comment:
We're not parsing `newImplementationLink` from the markdown anywhere currently so there's no way to trigger this code.  We'd need to add 
```
newImplementationLink: frontmatter.newImplementationLink || false
```
where we're defining `pageData` [in mdx.js (similar to your added code for `deprecated`, `beta`, and `demo`)](https://github.com/patternfly/patternfly-org/pull/3547/files#diff-97bd431987a2e7dda0f66ad7f07ac6cf4eeebf21fbe44a1b45ef905968aa14caR107-R109).